wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -340,6 +341,11 @@ public WorkspaceStatus getStatus(String workspaceId) {
           createInternalEnvironment(
               environment, workspaceConfig.getAttributes(), workspaceConfig.getCommands());
       RuntimeContext runtimeContext = infrastructure.prepare(runtimeId, internalEnv);
+
+      internalEnv.getMachines()
```

Comment:
What behavior would be if there was a machine without such an attribute?
What if the container doesn't have a corresponding machine?
Do you cover JWT proxy in this PR?

Reply:
> What behavior would be if there was a machine without such an attribute?

I think if machine without attribute - that's something that was bringing from our api, it's some internal stuff and we should not display it for end users. It's implementation specific stuff and it should be hidden.

> Do you cover JWT proxy in this PR?
I didn't take a look this case. I will take a look.

Reply:
@garagatyi Good catch. I think the following attribute should be added to machine of `JwtProxy`. See https://github.com/eclipse/che/blob/a1417ac9c63d40ffbbb5bffee46a55c36788c150/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/secure/jwtproxy/JwtProxyProvisioner.java#L256